src/panels/lovelace/cards/hui-shopping-list-card.ts

```diff
@@ -192,6 +190,9 @@ class HuiShoppingListCard extends LitElement implements LovelaceCard {
           padding: 9px 15px 11px 15px;
           cursor: pointer;
         }
+        paper-item-body {
+          width: 75%;
```

Comment:
This is putting a 75% on all items, both add and list items. Why not use the full width?

Reply:
Looked weird imo